moto/emr/models.py

```diff
@@ -359,6 +412,7 @@ def __init__(self, region_name):
         self.region_name = region_name
         self.clusters = {}
         self.instance_groups = {}
+        self.instances = []
```

Comment:
Doesn't look like this is used anywhere - we should be able to retrieve all instances by just iterating over the clusters, if we ever need a full list

Reply:
Removed it.